mentat/logging_config.py

```diff
@@ -43,6 +43,14 @@ def setup_logging():
     costs_logger.setLevel(logging.INFO)
     costs_logger.propagate = False
 
+    transcripts_logger = logging.getLogger("transcript")
+    transcripts_formatter = logging.Formatter("%(message)s")
+    transcripts_handler = logging.FileHandler(logs_path / "transcripts.log")
```

Comment:
```suggestion
    transcripts_handler = logging.FileHandler(logs_path / f"transcript_{timestamp}.log")
```

My one suggestion - makes it easier to review specific logs for debugging.